docs/user-guide/private-repositories.md

```diff
@@ -50,6 +50,9 @@ Then, connect the repository using any non-empty string as username and the acce
 > [!NOTE]
 > For some services, you might have to specify your account name as the username instead of any string.
 
+> [!NOTE]
+> For bitbucket, you have to specify your username as `x-token-auth`.
```

Comment:
```suggestion
> For BitBucket Cloud and BitBucket Data Center, you have to specify your username as `x-token-auth`.
```

Reply:
I did work with webhook stuff for bitbucket in the past but can't remember if there's anything apart from cloud and data center, like `bitbucket server`. If there doesn't exist one, you can simplify this as `BitBucket (Cloud or Data Center)` although this is a very minor tweak.